service/provider/src/main/java/gov/va/vro/service/provider/services/HealthEvidenceProcessor.java

```diff
@@ -22,12 +22,16 @@ public void process(Exchange exchange) {
     AbdEvidenceWithSummary evidence = exchange.getMessage().getBody(AbdEvidenceWithSummary.class);
 
     if (evidence.getErrorMessage() != null) {
-      log.error("Health Assessment Failed");
-      throw new MasException("Health Assessment Failed with error:" + evidence.getErrorMessage());
+      if ("insufficientHealthDataToOrderExam".equalsIgnoreCase(evidence.getErrorMessage())) {
```

Comment:
Instead of passing an error message from the assessment service, what if "null" was passed in as "sufficientForFastTracking" would we need to set the exchange property or would it already be set?

Reply:
Null can be passed as I changed the type of this field from primitive to Boolean object. The error message "insufficientHealthDataToOrderExam" is equal to Null being passed and is more verbose